lib/logthrdestdrv.h

```diff
@@ -59,6 +59,11 @@ typedef struct _LogThreadedDestWorker
 struct _LogThreadedDestDriver
 {
   LogDestDriver super;
+  struct iv_task  do_work;
+  struct iv_event wake_up_event;
```

Comment:
Just for curiousity: can you explain the guiding principle here?

I was thinking we are trying to move the frequent fields to the top for cpu-cache optimization?
I see do_work used a lot, but wake_up_event less, shutdown_event very rarely, while batch, processed_message, memory_usage, seq_num much more, but they are lower in the structure.

Reply:
I wanted do_work up there and the rest is not that important, so I've kept the grouping of ivykis events/tasks. 

This is probably just noise, so I will skip this patch.